.travis.yml

```diff
@@ -5,7 +5,7 @@ php:
   - 5.5
   - 5.6
   - 7.0
-  - hhvm
```

Comment:
Why removed? It should be upgrated to `trusty` as a main repo does:
https://github.com/yiisoft/yii2/blob/master/.travis.yml#L37-L51